### ISSUE TITLE
add installation of admission controller in rudr Docs 

### ISSUE THREAD

**Issue by wonderflow** [oam-dev/rudr#464]
The installation of [admission controller](https://github.com/oam-dev/admission-controller) should be part of rudr installation. So that could help rudr to mutate and validate spec applied to rudr.

**Comment by zhxu2** [comment]
Does the installation work on windows? It seems script for generating certs only works for linux environment

**Comment by wonderflow** [comment]
Yeah, current script for generating certs only works for linux, you can using existing certs on windows. Welcome to contribute a script for windows.
